_editions/2020/tasks/gamestory.md

```diff
@@ -5,44 +5,69 @@ year: 2020
 hide: true
```

Comment:
Is it ready to publish? when it's ready just change from `true` to `false` and request merge again :)

Reply:
Ah yes, didn't see the comment until now. I've made created another commit and set hidden to false :)